src/main/java/step2/service/LottoService.java

```diff
@@ -14,25 +14,25 @@
     private final Shop shop = new Shop();
     private final Judge judge = new Judge();
 
-    private final LottoGenerator lottoGenerator = new LottoGenerator();
-    private final MoneyGenerator moneyGenerator = new MoneyGenerator();
+    private final LottoMachine lottoGenerator = new LottoMachine();
+    private final MoneyMachine moneyGenerator = new MoneyMachine();
```

Comment:
```suggestion
    private final LottoMachine lottoMachine = new LottoMachine();
    private final MoneyMachine moneyMachine = new MoneyMachine();
```

클래스명을 리팩토링할 때는 이와 관련된 인스턴스 명도 함께 리팩토링해주세요!